package.nls.json

```
@@ -153,6 +153,7 @@
     "vscode-docker.config.template.composeDown.match": "The regular expression for choosing the right template. Checked against docker-compose YAML files, folder name, etc.",
     "vscode-docker.config.template.composeDown.description": "Command templates for `docker-compose down` commands.",
     "vscode-docker.config.docker.containers.groupBy": "The property to use to group containers in Docker view: ContainerId, ContainerName, CreatedTime, FullTag, ImageId, Networks, Ports, Registry, Repository, RepositoryName, RepositoryNameAndTag, State, Status, Tag, or None",
+    "vscode-docker.config.docker.containers.groupByLabel": "The items will be grouped by the value of this Docker label",
```

```suggestion
    "vscode-docker.config.docker.containers.groupByLabel": "The items will be grouped by the value of this container label",
```
More generic and equally or more accurate, IMO.

---

Agreed